media-tv/kodi/kodi-21.2.ebuild

```diff
@@ -166,10 +166,7 @@ COMMON_TARGET_DEPEND="${PYTHON_DEPS}
 		x11-libs/libxkbcommon
 	)
 	gles? (
-		|| (
-			>=media-libs/mesa-24.1.0_rc1[opengl]
-			<media-libs/mesa-24.1.0_rc1[gles2]
-		)
+		>=media-libs/mesa-24.1.0_rc1[opengl]
```

Comment:
There are only versions >= 24.1.7 in portage. I think we should unconditionally depend on media-libs/mesa[opengl]. From my understanding, Kodi requires either the OpenGL backend or the GLES backend, and now both need the opengl flag for mesa.

https://github.com/kodi-overlay/kodi-overlay/blob/main/media-tv/kodi/kodi-21.2.ebuild#L131

Reply:
Ok, this sounds reasonable. Do you want attribution? Via me taking the patch from kodi-overlay and applying it via git-am.

Reply:
Not necessarily, just the simplest and fastest way for you.